docs/tutorials/jokes.md

```diff
@@ -90,21 +90,16 @@ npx create-remix@latest
 
 <docs-info>
 
-This may ask you whether you want to install `create-remix` to run the command. Enter `y`. It will only be installed temporarily to run the setup script.
+This may ask you whether you want to install the following packages `create-remix@latest`. To run the command, enter `y`. It will only be installed the first time to run the setup script.
```

Comment:
```suggestion
This may ask you whether you want to install `create-remix@latest`. Enter `y`. It will only be installed the first time to run the setup script.
```